src/FOSHttpCacheBundle.php

```diff
@@ -30,17 +29,10 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new LoggerPass());
         $container->addCompilerPass(new TagListenerPass());
         $container->addCompilerPass(new HashGeneratorPass());
-        if (version_compare(Kernel::VERSION, '3.4', '>=')
-            && version_compare(Kernel::VERSION, '4.1', '<')
```

Comment:
this was a regression we introduced at some point. the compiler pass is still relevant with newer symfony versions, as it tells the UserContextListener whether to set a magic header for the symfony core session listener.